### ISSUE TITLE
Error compiling dev-qt/qtstyleplugins-20170311-r1

### ISSUE THREAD

**Issue by kumarayush2104** [zhtengw/deepin-overlay#86]
Error :-

https://pasteboard.co/JbxDakq.jpg

**Comment by tmu-sprd** [comment]
As a workaround, mask qt-5.15 until fixed:
`/etc/portage/package.mask`:
```
>=dev-qt/qtcore-5.15.0
>=dev-qt/qtxcb-private-headers-5.15.0
>=dev-qt/qtvirtualkeyboard-5.15.0
>=dev-qt/qtdeclarative-5.15.0
>=dev-qt/qtgui-5.15.0
>=dev-qt/designer-5.15.0
>=dev-qt/qtquickcontrols-5.15.0
>=dev-qt/linguist-tools-5.15.0
>=dev-qt/qtopengl-5.15.0
>=dev-qt/qdbus-5.15.0
>=dev-qt/qtcharts-5.15.0
>=dev-qt/qttest-5.15.0
>=dev-qt/qtwayland-5.15.0
>=dev-qt/qtmultimedia-5.15.0
>=dev-qt/qtsensors-5.15.0
>=dev-qt/qtimageformats-5.15.0
>=dev-qt/qtsql-5.15.0
>=dev-qt/qtwidgets-5.15.0
>=dev-qt/qtxmlpatterns-5.15.0
>=dev-qt/qtx11extras-5.15.0
>=dev-qt/qtwebchannel-5.15.0
>=dev-qt/qtnetwork-5.15.0
>=dev-qt/qtgraphicaleffects-5.15.0
>=dev-qt/qtquickcontrols2-5.15.0
>=dev-qt/qtxml-5.15.0
>=dev-qt/qtsvg-5.15.0
>=dev-qt/qtscript-5.15.0
>=dev-qt/qtprintsupport-5.15.0
>=dev-qt/qtdbus-5.15.0
>=dev-qt/qtconcurrent-5.15.0
```

**Comment by kumarayush2104** [comment]
thanks, I will try that, can you ping me in this issue when it's fixed, thanks again..😊

**Comment by zhtengw** [comment]
dev-qt/qtstyleplugins is not needed by dde-base/dde-qt5integration any more, masked for removal. https://github.com/zhtengw/deepin-overlay/commit/0a6c002a1db392b4dcb4bbc898582334e591c7cf

**Comment by kumarayush2104** [comment]
build with qt-5.15 is fixed now ??

**Comment by zhtengw** [comment]
> 
> 
> build with qt-5.15 is fixed now ??

Fixed https://github.com/zhtengw/deepin-overlay/commit/58f61c37f190fb7df0bb77c962403776c2a41404.

**Comment by kumarayush2104** [comment]
dde-base/dde-network-utils-5.1.0101.2 cannot be compiled

error :- http://dpaste.com/23210Q5

**Comment by zhtengw** [comment]
> 
> 
> dde-base/dde-network-utils-5.1.0101.2 cannot be compiled
> 
> error :- http://dpaste.com/23210Q5

Missing dependency description of x11-libs/gsettings-qt, fixed now.

**Comment by kumarayush2104** [comment]
error compiling dev-qt/dde-qt5platform-plugins-9999_r1

error :- http://dpaste.com/0QEJ9S5

**Comment by zhtengw** [comment]
> 
> 
> error compiling dev-qt/dde-qt5platform-plugins-9999_r1
> 
> error :- http://dpaste.com/0QEJ9S5

```
fatal: unable to access 'https://github.com/linuxdeepin/qt5platform-plugins.git/': Couldn't resolve host 'github.com'
```
Please resolve your network issue first.

**Comment by kumarayush2104** [comment]
oops sorry, however a new issue

media-sound/deepin-voice-recorder-5.0.0

error:- http://dpaste.com/3HRZGN8

media-gfx/deepin-image-viewer-5.6.3.2

error:- http://dpaste.com/14W99H5

**Comment by kumarayush2104** [comment]
sir, some apps are still not compiling with qt-5.15

**Comment by zhtengw** [comment]
@kumarayush2104 https://github.com/zhtengw/deepin-overlay/commit/804a52d363300c6bee5dbced04276920053f3294 Fix some multimedia packages build with qt 5.15.

BTW, media-sound/deepin-voice-recorder is not maintained by upstream now, please install dde-extra/deepin-voice-note instead.

**Comment by LinuxUserGD** [comment]
@zhtengw For me those packages seems to still fail to compile:
```Swift
sudo emerge --ask dde-meta
[ebuild  NS    ] dde-base/dtkwidget-2.1.1 [5.2.0-r1]
[ebuild  N     ] dde-extra/deepin-clone-5.0.3  USE="gui" 
[ebuild  N     ] dde-extra/deepin-topbar-0.6.6-r1  USE="albert" 
[ebuild  N     ] dde-base/dde-control-center-5.2.0.0  USE="redshift systemd" 
[ebuild  N     ] dde-extra/dde-meta-apps-20 
[ebuild  N     ] dde-base/dde-meta-20  USE="extra grub kwin manual multimedia plymouth (policykit) screensaver systemd terminal turbo (-elogind) -mutter"
```

**Comment by zhtengw** [comment]
@LinuxUserGD Fix dtkwidget-2.1.1 and deepin-clone-5.0.3 with https://github.com/zhtengw/deepin-overlay/commit/d32550de0ec9877211ee8e1e9ac9ee334434dcce. Please try again.
If you meet any other compile error, feel free to open a new issue and upload your build log.

**Comment by LinuxUserGD** [comment]
Thanks, those are fixed. Opened a new one for `dde-control-center`
https://github.com/zhtengw/deepin-overlay/issues/91
